guardian/frontend
# Add fetchEmail callback to Braze components

## What does this change?
This update mirrors the update made in DCR to pipe a function to get the email of a signed-in user into Braze components. However, as very few Braze banners now appear on Frontend-served fronts, in this PR the function (a Promise) will return `null` rather than do the work to seek out a signed-in user's email address.

## How to test
See https://github.com/guardian/dotcom-rendering/pull/8561 for details on how we intend to test this update
